ucx.spec

```
@@ -121,8 +122,8 @@ Provides header files and examples for developing with UCX.
 make %{?_smp_mflags} V=1
 
 %install
-make DESTDIR=%{buildroot} install
-rm -f %{buildroot}%{_libdir}/*.la
+make DESTDIR=%{buildroot} install prefix=%{_prefix} libdir=%{_libdir} datadir=%{_datadir}
+rm -f %{buildroot}%{_libdir}/*la
```

It really is only `.la` files we want to remove though, right?

```suggestion
rm -f %{buildroot}%{_libdir}/*.la
```